src/test/java/org/sagebionetworks/bridge/reporter/worker/RetentionReportGeneratorTest.java

```diff
@@ -38,8 +40,14 @@
     
     private static final DateTime STUDY_START_DATE_1 = DateTime.parse("2017-06-05T00:00:00.000Z");
     private static final DateTime STUDY_START_DATE_2 = DateTime.parse("2017-06-04T00:00:00.000Z");
+    
     private static final DateTime SIGN_IN_ON = DateTime.parse("2017-06-09T00:00:00.000Z");
-    private static final DateTime UPLOADED_ON = DateTime.parse("2017-06-09T00:00:00.000Z");
+    private static final DateTime SIGN_IN_ON_INVALID = DateTime.parse("2016-06-09T00:00:00.000Z");
+    private static final DateTime SIGN_IN_ON_TIMEZONE = DateTime.parse("2017-06-04T06:50:21.650-07:00");
```

Comment:
6:50-0700 is 13:50Z. We should pick something later in the day that will actually change the date to verify that our timestamp logic is working correctly.